services/logging/src/test/logging.controller.test.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { LogLoggedMessage } from '@carbon-platform/api/logging'
+
+import { LogDnaService } from '../main/log-dna.service'
+import { LoggingController } from '../main/logging.controller'
+
+test('logLogged runs without crashing', async () => {
+  const logDnaService = new LogDnaService()
+  const loggingController = new LoggingController(logDnaService)
+
+  const data: LogLoggedMessage = {
+    component: 'test',
+    environment: 'test',
+    level: 'debug',
+    message: 'test message',
+    service: 'test service',
+    timestamp: Date.now()
+  }
+
+  await expect(loggingController.logLogged(data)).resolves.not.toThrow()
+})
+
+describe('message validation', () => {
+  let logDnaService: LogDnaService
+  let loggingController: LoggingController
+  let data: LogLoggedMessage
+
+  beforeEach(() => {
+    logDnaService = new LogDnaService()
+    loggingController = new LoggingController(logDnaService)
+    ;(loggingController as any).nestLogger = {
```

Comment:
That would definitely be more clear as far as the intent of that part of the test, but unfortunately since `nestLogger` is marked as read-only in `PlatformController`, you can't set it when the object is asserted as a `PlatformController`